driver/normalizer/normalizer.go

```diff
@@ -88,8 +110,8 @@ func mapStr(nativeType string) Mapping {
 			{Name: uast.KeyType, Op: String("Boxed" + nativeType)},
 			{Name: "boxed_value", Op: UASTType(uast.String{}, Obj{
 				uast.KeyPos: Var("pos_"),
-				"Value": Var("s"),
-				"Format": String(""),
+				"Value":     Var("s"),
+				"Format":    String(""),
```

Comment:
Not related to the change, but zero values may be omitted.

Reply:
Done in next commit.